src/Resources/config/services.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2014-2020 Spomky-Labs
```

Comment:
```diff
- Copyright (c) 2014-2020 Spomky-Labs
+ Copyright (c) 2021 Yoan Bernabeu
```

Reply:
this may sounds harsh : you can use @author annotation if you want to specify the ownership of this class :)